crates/rattler_repodata_gateway/src/gateway/builder.rs

```diff
@@ -85,13 +86,16 @@ impl GatewayBuilder {
                 .join("rattler/cache")
         });
 
+        let package_cache = PackageCache::new(cache.clone());
```

Comment:
Would be nice to add a `with_package_cache` function to the builder.

Reply:
Added it, as an optional so it is possible to use default.